modules/subscriptions.php

```diff
@@ -852,6 +852,11 @@ function widget( $args, $instance ) {
 		$show_subscribers_total = (bool) $instance['show_subscribers_total'];
 		$subscribers_total      = $this->fetch_subscriber_count(); // Only used for the shortcode [total-subscribers]
 
+		if ( $instance['show_only_email_and_button'] ) {
```

Comment:
Why do we want to remove that information from the block? Wouldn't it be nice if folks could opt to use or customize that information?

Reply:
This allows the published post/form to conform to @MichaelArestad 's design, https://wp.me/pafL3P-4w-p2

The idea was that anyone who wants to add things to their subscription form could simply add other block types above, below, to the side of it, etc. and include anything they want.